ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -157,6 +152,17 @@ module.exports = Marionette.LayoutView.extend({
             }
         }
     },
+    handlePolygonDeserialization(filter) {
+        const polygonArray = (filter.value && filter.value.value && CQLUtils.arrayFromPolygonWkt(filter.value.value)) || [];
+        const bufferWidth = filter.polygonBufferWidth || filter.distance;
+
+        this.model.set({
+            mode: 'poly',
+            polygon: filter.polygon || polygonArray,
```

Comment:
❓ Any reason not to do this above where we're determining polygonArray?

Reply:
I think it would be unreadable if I added another `||` up there.